pkgs/misc/jackaudio/default.nix

```diff
@@ -17,12 +16,10 @@ let
   inherit (python2Packages) python dbus-python;
   shouldUsePkg = pkg: if pkg != null && pkg.meta.available then pkg else null;
 
-  libOnly = prefix == "lib";
-
   optDbus = if stdenv.isDarwin then null else shouldUsePkg dbus;
-  optPythonDBus = if libOnly then null else shouldUsePkg dbus-python;
-  optLibffado = if libOnly then null else shouldUsePkg libffado;
-  optAlsaLib = if libOnly then null else shouldUsePkg alsaLib;
+  optPythonDBus = shouldUsePkg dbus-python;
```

Comment:
You can probably get rid of shouldUsePkg. I think they are officially deprecated (just use dbus-python directly).

Reply:
It don't used outside of mixer, so if we move mixer to own app, we can drop dbus-python in main ffado.